src/Language/Parser.php

```diff
@@ -61,6 +61,25 @@ class Parser implements ParserInterface
      */
     protected $lexer;
 
+    /**
```

Comment:
I would suggest adding a comment that outlines the purpose of this magic function.